packages/dpkg.rb

```diff
@@ -53,9 +40,13 @@ def self.install
     Dir.chdir ("#{CREW_DEST_PREFIX}/bin") do
       system 'mv dpkg dpkg-run'
       system "echo '#!/bin/bash' > dpkg"
-      system "echo 'dpkg-run --force-depends --root=#{CREW_PREFIX} \"$@\"' >> dpkg"
+      system "echo 'dpkg-run --force-not-root --force-depends --root=#{CREW_PREFIX} \"$@\"' >> dpkg"
       system 'chmod a+x dpkg'
     end
+    Dir.chdir ("#{CREW_DEST_PREFIX}") do
+      puts "This will make dpkg work without root privileges".lightred
+      system "sudo", "chown", "-R", "chronos:chronos", "./var/lib/dpkg"
```

Comment:
Instead of `chronos`, please abstract with `#{USER}` instead.  See `lib/const.rb`.